src/hydat/parse.ts

```diff
@@ -1,26 +1,18 @@
-export function isAlpha(c: string) {
-  const regex = new RegExp('^[A-Za-z]$');
-  return regex.test(c);
-}
+const isAlpha = RegExp.prototype.test.bind(/^[A-Za-z]$/);
+const isDigit = RegExp.prototype.test.bind(/^[0-9]$/);
```

Comment:
結果的にこっちのほうが文字数少ないしわかりやすいかと
```suggestion
const isAlpha = (c)=>/^[A-Za-z]$/.test(c);
const isDigit = (c)=>/^[0-9]$/.test(c);
```

Reply:
おおおおおおお